python/ray/exceptions.py

```diff
@@ -429,10 +429,7 @@ class RuntimeEnvSetupError(RayError):
     """Raised when a runtime environment fails to be set up."""
 
     def __str__(self):
-        return (
-            "The runtime environment for this task or actor failed to be "
-            "installed. Corresponding error logs should have been streamed "
-            "to the driver's STDOUT.")
+        return "The runtime_env failed to be set up."
```

Comment:
@edoakes Just clarifying: the change here is unrelated to the main PR, but you would like to simplify this err msg in general, right?

Reply:
Yes, the error message wasn't always correct and would cause confusion for job submission. @rkooo567 has a task to make this much more relevant (including the underlying error).

Reply:
I believe this error was also introduced since last release :)